pepper-apis/studybuilder-cli/studies/singular/patches/ddp-8554-dependent-validations.conf

```diff
@@ -0,0 +1,41 @@
+{
+  "validations": [
+    # When using the Enroll My Adult Dependent flow, if a user enters an age at or above the AOM for their region in
+    # the pre-qual but enters a date of birth that equates to <AOM for their region in the adult dependent consent,
+    # an error message should appear under the consent DOB field.
+    # Formally: PREQUAL.AGE > 18 && CONSENT.AGE < 18
+    {
+      "messageTemplate": {
+        "templateType": "HTML",
+        "templateText": "$singular_consent_dependent_validation",
+        "variables": [
+          {
+            "name": "singular_consent_dependent_validation",
+            "translations": [
+              {
+                "language": "en",
+                "text": """Please check the date of birth you have entered. If the date is correct, then your dependent
+                           is younger than the age required by local law to self-enroll in this study. In that case,
+                           please return to the dashboard and click "Enroll my child" to begin a new enrollment for
+                           your dependent, or contact us for help at (650) 561-6750 or
+                           <a href="mailto:contact@projectsingular.org">contact@projectsingular.org</a>.
+                        """
+              }
+            ]
+          }
+        ]
+      },
+      "stableIds": ["CONSENT_DEPENDENT_DATE_OF_BIRTH", "ENROLLING_DEPENDENT_AGE"],
+      "precondition": """
+                          user.studies["singular"].forms["ADD_PARTICIPANT_DEPENDENT"].questions["ENROLLING_DEPENDENT_AGE"].isAnswered()
+                          &&
+                          user.studies["singular"].forms["CONSENT_DEPENDENT"].questions["CONSENT_DEPENDENT_DATE_OF_BIRTH"].isAnswered()
+                          """,
+      "expression": """
+                        user.studies["singular"].forms["ADD_PARTICIPANT_DEPENDENT"].questions["ENROLLING_DEPENDENT_AGE"].answers.value() > 18
```

Comment:
I believe 18 years is considered adult. Check should probably be >=

Reply:
Done